dns-server/src/bin/dnsadm.rs

```diff
@@ -115,9 +115,12 @@ async fn main() -> Result<()> {
             for zone_config in &config.zones {
                 println!("\nzone {:?}", zone_config.zone_name);
 
-                for kv in &zone_config.records {
-                    println!("    key {:?}:", kv.key.name);
-                    for record in &kv.records {
+                // Sort the records so that we get consistent ordering.
+                let records: BTreeMap<_, _> =
```

Comment:
Should we store them in a `BTreeMap` to begin with so they're always in a predictable order?

Reply:
Yeah, I started with BTreeMap, but the progenitor-generated client uses a HashMap, and this is not configurable.